extensions/pkg/controller/healthcheck/controller.go

```diff
@@ -161,11 +155,21 @@ func add(mgr manager.Manager, args AddArgs) error {
 	if err != nil {
 		return err
 	}
+
+	log.Log.Info("Registered health check controller", "Kind", args.registeredExtension.groupVersionKind.Kind, "type", args.Type, "health check type", args.registeredExtension.healthConditionTypes, "sync period", args.SyncPeriod.Duration.String())
```

Comment:
Nit

```suggestion
	log.Log.Info("Registered health check controller", "kind", args.registeredExtension.groupVersionKind.Kind, "type", args.Type, "health check type", args.registeredExtension.healthConditionTypes, "sync period", args.SyncPeriod.Duration.String())
```

Ref https://github.com/kubernetes-sigs/controller-runtime/blob/master/TMP-LOGGING.md#logging-structured-values-key-value-pairs